fvm/state/accounts.go

```diff
@@ -159,27 +155,10 @@ func (a *Accounts) SetPublicKeyCount(address flow.Address, count uint64) {
 }
 
 func (a *Accounts) GetPublicKeys(address flow.Address) (publicKeys []flow.AccountPublicKey, err error) {
-	var countBytes []byte
-	countBytes, err = a.ledger.Get(
-		string(address.Bytes()), string(address.Bytes()), keyPublicKeyCount,
-	)
-	if err != nil {
-		return nil, newLedgerGetError(keyPublicKeyCount, address, err)
-	}
-
-	var count uint64
 
-	if countBytes == nil {
-		count = 0
-	} else {
-		countInt := new(big.Int).SetBytes(countBytes)
-		if !countInt.IsUint64() {
-			return nil, fmt.Errorf(
-				"retrieved public key account count bytes (hex-encoded): %x do not represent valid uint64",
-				countBytes,
-			)
-		}
-		count = countInt.Uint64()
+	count, err := a.GetPublicKeyCount(address)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get public key count of account: %w", err)
```

Comment:
Thanks Bastian for cleaning this, I always wondered what makes this code different than `GetPublicKeyCount` :)